platform/linux-generic/m4/odp_zlib.m4

```diff
@@ -0,0 +1,10 @@
+#########################################################################
+# Check for libz availability
+#########################################################################
+
+AC_CHECK_HEADERS([zlib.h])
+
+ZLIB_LIBS="-lz"
```

Comment:
DEPENDANCIES file also needs to be updated.

Reply:
Fixed in v3.